lib/_ffi/js/dom.scm

```diff
@@ -7,25 +7,25 @@
 
 ;;;============================================================================
 
-(define (select selector)
+(define (querySelector selector)
```

Comment:
The naming is not consistent.  It should be called `document.querySelector` and there should be a seperate procedure `(querySelector elem selector)`.  Same for `getElementById` and `createElement`.  It might make sense to have an exported `document` variable that is initialized to `\document` so this can be done: `(createElement document "div")`.